packages/web/src/pages/upload-page/fields/AccessAndSaleMenuFields.tsx

```diff
@@ -37,8 +38,11 @@ const messages = {
     'This track is marked as a remix. To enable additional availability options, unmark within Remix Settings.',
   done: 'Done',
   public: 'Public (Free to Stream)',
-  publicSubtitle:
-    'Public tracks are visible to all users and appear throughout Audius.',
+  publicSubtitle: (contentType: 'album' | 'track') =>
+    `Public ${pluralize(
+      contentType,
+      2
+    )} are visible to all users and appear throughout Audius.`,
```

Comment:
calling pluralize maybe overkill? idk

Reply:
I would agree. I feel like it probably is given we know the words are only `album`/`track`. Adding an s is probably simpler